### DISCUSSION TITLE
Koha-Suomen asiantuntijaryhmän muistio 4/24 julkaistu

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#1123] [Tiedotteet]
Koha-Suomen asiantuntijaryhmä kokoontui ma 18.3.2024 ja kokouksen muistio on nyt julkaistu Koha-Suomen verkkosivuilla Muistiot-osiossa.

[Koha-Suomen asiantuntijaryhmän muistio 4/2024](https://koha-suomi.fi/asiantuntijaryhma2024#asiantuntijaryhm%C3%A4n-muistio-424)

**Comment by LastuLauriH** [comment]
"kokoontui ma 19.3.2024"
18.3. :)

**Reply by AnneliO** [reply]
No huppista, perinteinen päivämäärävirhe. Nyt korjattu. Kiitos hoksautuksesta! :)
